README.md

```diff
@@ -57,3 +57,42 @@ To create a sawtooth signal starting at 0dB increasing in 2dB steps every 50 mic
 "sudo attenuator_lab_brick -ramp -start 0 -end 60 -step 2 -t 50 us -rr 8"
 ```
 For a more enhanced usage example of our tool within your wireless experiments look at the "run-experiment.sh" shell script.
+
+## More (specific) usage examples
+Set 40dB attenuation to single device (automatically detected) for 20 seconds
+
+"sudo attenuator_lab_brick s -a 40 -t 20"
```

Comment:
The commands should be enclosed with code blocks as used in the rest of the README file (\`\`\`\n command \n\`\`\`)

Reply:
Hi @ikstream,
do you want me to do that change or you are working on that?

Reply:
Would be great if you could edit it by yourself

Reply:
Ok, I've just pushed some changes to the README file.
Let me know if now it is ok or if you need some other changes.
Cheers!